samwise.js

```diff
@@ -4,32 +4,29 @@
  * Copyright 2016 Henrik Ingo, henrik.ingo@avoinelama.fi
```

Comment:
SCRIPT-I like how you've used promise chaining here, it makes the code flow very clear.